evalai/utils/common.py

```diff
@@ -201,20 +220,21 @@ def upload_file_using_presigned_url(challenge_phase_pk, file, file_type, submiss
 
         data = {
             "parts": json.dumps(response.get("parts")),
-            "upload_id": upload_id
+            "upload_id": upload_id,
+            "annotations_uploaded_using_cli": True,
```

Comment:
Can we please add this in data dict `"annotations_uploaded_using_cli": True,` only when annotations are uploaded?

Reply:
@RishabhJain2018 Updated the code.